src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -187,6 +188,9 @@ public function load(array $configs, ContainerBuilder $container)
             if (!class_exists(BaseYamlLintCommand::class)) {
                 $container->removeDefinition('console.command.yaml_lint');
             }
+            if (!class_exists(AbstractUid::class)) {
```

Comment:
```suggestion
            if (!class_exists(UidGenerateCommand::class)) {
```

💡  Now that the command has moved, we can check for the actual command class instead.

Reply:
True, fixed.